pkg/doc/verifiable/credential.go

```diff
@@ -655,21 +786,46 @@ func (vc *Credential) JWT(signatureAlg JWTAlgorithm, privateKey interface{}, key
 	// validate all ok, sign with the RSA key, and return a compact JWT
 	rawJWT, err := builder.CompactSerialize()
 	if err != nil {
-		return "", fmt.Errorf("failed to sign JWT")
+		return "", fmt.Errorf("failed to sign JWT: %w", err)
 	}
 
 	return rawJWT, nil
 }
 
-func (vc *Credential) getFirstSubjectID() string {
+// FirstSubjectID gets ID of first subject if present or return empty string otherwise
+func (vc *Credential) FirstSubjectID() string {
```

Comment:
This function shouldn't exist at all. If the JSON-LD VC contains multiple subjects, the `JWT()` method should return an error indicating as such.

Reply:
Agree! I will fix that.